styles/main.css

```diff
@@ -0,0 +1,167 @@
+* {
+    box-sizing: border-box;
+    font-family: "Pretendard Variable";
+}
+
+header {
+    padding: 0 200px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.btn {
+    background-color: #3692FF;
+    color: #F3F4F6;
+    font-weight: 600;
+    text-align: center;
+    text-decoration: none;
+}
+
+h2 {
+    color: #374151;
+    font-size: 40px;  /* 40px */
+    font-weight: 700;
+    line-height: 56px;
+    margin: 0;
+}
```

Comment:
## `font-size` 는 `em`을 고려해볼 수 있습니다.
### 다음은 `stackoverflow`의 `px vs em`과 관련된 내용 중 일부 번역본입니다 :
[제가 이 질문을 한 이유는 제가 CSS에서 즐겁게 해킹을 하던 시절에 em을 사용하는 방법을 잊었기 때문입니다. 사람들은 제가 글꼴 크기 조정에 대해 이야기하지 않았기 때문에 질문을 일반적으로 유지했다는 것을 알아차리지 못했습니다. 저는 페이지의 주어진 블록 요소 에 스타일을 정의하는 방법에 더 관심이 있었습니다.

[Henrik Paul](https://stackoverflow.com/questions/609517/why-em-instead-of-px/609541#609541) 과 다른 사람들이 지적 했듯이 em은 요소에 사용된 글꼴 크기에 비례합니다. 블록 요소의 크기를 px로 정의하는 것이 일반적인 관행이지만, 브라우저에서 글꼴 크기를 조정하면 일반적으로 이 디자인이 깨집니다. 글꼴 크기 조정은 일반적으로 단축키 Ctrl+ +또는 Ctrl+ 로 수행됩니다 -. 따라서 좋은 관행은 대신 em을 사용하는 것입니다.

px를 사용하여 너비 정의
다음은 설명적인 예입니다. 스타일리시한 날짜 상자로 바꾸고 싶은 div 태그가 있다고 가정하면, 다음과 같은 HTML 코드가 있을 수 있습니다.
```html
<div class="date-box">
    <p class="month">July</p>
    <p class="day">4</p>
</div>
```
간단한 구현은 클래스의 너비를 date-boxpx로 정의하는 것입니다.

```css
* { margin: 0; padding: 0; }

p.month { font-size: 10pt; }

p.day { font-size: 24pt; font-weight: bold; }

div.date-box {
    background-color: #DD2222;
    font-family: Arial, sans-serif;
    color: white;
    width: 50px;
}
```
### 문제
하지만 브라우저에서 텍스트 크기를 키우고 싶다면 디자인이 깨질 것입니다. 텍스트는 상자 밖으로도 흘러내리는데, 이는 [flodin이](https://stackoverflow.com/questions/609517/why-em-instead-of-px/609543#609543) 지적한 대로 SO의 디자인에서 일어나는 일과 거의 같습니다. 상자가 잠겨 있기 때문에 너비가 같은 크기로 유지되기 때문입니다 50px.

대신 em을 사용하세요
더 똑똑한 방법은 대신 ems로 너비를 정의하는 것입니다.
```css
div.date-box {
    background-color: #DD2222;
    font-family: Arial, sans-serif;
    color: white;
    width: 2.5em;
}

* { margin: 0; padding: 0; font-size: 10pt; }

// Initial width of date-box = 10 pt x 2.5 em = 25 pt
// Will also work if you used px instead of pt
```
이렇게 하면 날짜 상자에 유동적인 디자인이 생깁니다. 즉, 상자는 날짜 상자에 정의된 글꼴 ​​크기에 비례하여 텍스트와 함께 커집니다. 이 예에서 글꼴 크기는 *10pt로 정의되어 있고 해당 글꼴 크기의 2.5배로 커집니다. 따라서 브라우저에서 글꼴 크기를 조정할 때 상자는 해당 글꼴 크기의 2.5배가 됩니다.](https://stackoverflow.com/questions/609517/why-em-instead-of-px)

[원본 보기](https://stackoverflow.com/questions/609517/why-em-instead-of-px)